package.json

```diff
@@ -25,34 +25,35 @@
   },
   "homepage": "https://github.com/emartech/angular-translate#readme",
   "engines": {
-    "node": "10.16.3"
+    "node": ">=10.18.0 <13.0.0",
+    "yarn": ">=1.22.4 <2"
   },
   "dependencies": {
-    "@angular/cli": "8.3.17",
+    "@angular/cli": "8.3.28",
     "flat": "5.0.0",
-    "ramda": "0.26.1",
+    "ramda": "0.27.0",
```

Comment:
btw can we remove this? this is the only place where we use it and if I'm right we could refactor it easily: https://github.com/emartech/angular-translate/blob/0054a67ace9eea0653f8e4c532c5fde8fa51a594/lib/service.ts#L60

Reply:
like here: https://github.com/emartech/angular-translate/blob/0054a67ace9eea0653f8e4c532c5fde8fa51a594/lib/service.ts#L30

Reply:
we can try